packages/ember-routing-views/lib/components/link-to.js

```diff
@@ -769,7 +769,7 @@ let LinkComponent = EmberComponent.extend({
     this.set('queryParams', queryParams);
 
     // 4. Any remaining indices (excepting `targetRouteName` at 0) are `models`.
-    let models = [];
+    let models = new Array(params.length - 1);
```

Comment:
This seems to be throwing an error when `params` is empty.

![screenshot](https://monosnap.com/file/LAEpjEU8BM7o5nBbigXvJgL0I6BkLt.png)